### PR TITLE
interfaces/udev: do not call udev when in preseed mode

### PR DESCRIPTION
**Pull request by bboozzoo**
The udev backend was not preseed aware and would always call udev reload, even before introducing #13642 and the errors were masked. Check if the backend is running in preseed mode and do not call udev at all.

### CODE REVIEW

File: interfaces/udev/backend.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func (b *Backend) Setup(appSet *interfaces.SnapAppSet, opts interfaces.Confineme
 		}
 	}
 
-	if needReload {
+	if needReload && !b.preseed {
```
Comment (andrewphelpsj) [inline interfaces/udev/backend.go:143]: Maybe a comment explaining why we don't go in here if in preseed mode?

**Review by andrewphelpsj — APPROVED** [review]
Thank you!

**Review by pedronis — APPROVED** [review]
thanks

**Review by pedronis — APPROVED** [review]
still ok

**Review by pedronis — APPROVED** [review]
+1 to last change

**Pull request merged**